.gitignore

```diff
@@ -3,10 +3,8 @@ dist/
 *.d
 private/
 *.properties
-*.xml
 *.mk
 *.bash
-Makefile
```

Comment:
And by copy and pasting, I mean if you move the project folder into the D:/ for example, it won't work.